numba/parfor.py

```diff
@@ -3131,7 +3134,8 @@ def parfor_typeinfer(parfor, typeinferer):
     # assigned to a tuple from individual indices
     first_block = min(blocks.keys())
     loc = blocks[first_block].loc
-    index_assigns = [ir.Assign(ir.Const(1, loc), v, loc) for v in index_vars]
+    # XXX
+    index_assigns = [ir.Assign(ir.Const(1, loc=loc, use_literal_type=False), v, loc) for v in index_vars]
```

Comment:
The `XXX` comment is for the `use_literal_type=False`.  I'll open an issue about it.